main.go

```diff
@@ -384,7 +388,12 @@ func makeServiceChanges(cf *Cloudflare, kubeClientset *kubernetes.Clientset, ser
 		service.Annotations[annotationCloudflareState] = string(cloudflareStateByteArray)
 
 		// update service, because the state annotations have changed
-		service, err = kubeClientset.CoreV1().Services(service.Namespace).Update(service)
+		service, err = kubeClientset.CoreV1().Services(service.Namespace).Update(ctx, service, metav1.UpdateOptions{
```

Comment:
if using default options then can just have it blank `metav1.UpdateOptions{}`

Reply:
You are right. The vim pluggin auto expanded those on save. Reverting those to the empty structs.